features/step_definitions/step_eight_steps.rb

```diff
@@ -53,8 +44,8 @@ def group_eight(index)
 end
 
 Then(/^I should see an income list for myself and my partner:$/) do |incomes|
-  expect(group_eight(0).h3.text).to eq 'Your income'
-  expect(group_eight(1).h3.text).to eq 'Your partner\'s income'
+  expect(group_eight(0)).to have_your_income
```

Comment:
What does have_your_income mean ?  I cant see this defined anywhere ?

Reply:
That'll be in the page object 'element :your_income, 'h3', text: 'Your income''